01-HTML-CSS-Basics/1. HTML/Punto 1/i. HolaMundo.html

```diff
@@ -0,0 +1,3 @@
+<head>
```

Comment:
You need to put the whole html structure: doctype, html tag, head tag and body tag.